### DISCUSSION TITLE
./mvnw package -Pagent,dist  failed

### DISCUSSION THREAD

**Discussion by bongmu** [apache/skywalking#7558] [Q&A]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Bug

___
### Question
I tried to rebuild the agent package locally, but an error was reported
___
### Bug
- Which version of SkyWalking, OS, and JRE?
- OS version: centos 7 
- JDK: 1.8
- maven: 3
![image](https://user-images.githubusercontent.com/33013978/130717026-43b3598b-a442-4a7e-b653-04b20aaad1bd.png)

___
### Requirement or improvement
- Please describe your requirements or improvement suggestions.

**Comment by wu-sheng** [comment]
`./mvnw package -Pagent,dist` only works at least you have one time `./mvnw package`. The webapp and oap-server binary packages are required existing locally.

**Reply by bongmu** [reply]
The webapp and oap-server binary packages are already existing locally

error  ,  how do it？

**Comment by bongmu** [comment]
![image](https://user-images.githubusercontent.com/33013978/130717938-b867e8f0-dd7e-4dbb-99c3-3f7fed23755d.png)
Please see the picture


The webapp and oap-server binary packages are already existing locally

**Reply by wu-sheng** [reply]
I don't mean the folder. Read the path in your logs. It is very clear.

**Reply by lizhiyoumai** [reply]
I do like this before, it's ok.But now, it doesn't work, how can I fix this error?

**Comment by bongmu** [comment]
I downloaded the latest version of the source code and rebuilt it locally
Download link:  https://www.apache.org/dyn/closer.cgi/skywalking/8.7.0/apache-skywalking-apm-8.7.0-src.tgz
